cmake/onnxruntime_python.cmake

```diff
@@ -157,19 +157,21 @@ if (onnxruntime_ENABLE_EAGER_MODE OR onnxruntime_ENABLE_LAZY_TENSOR)
     "${REPO_ROOT}/cmake/external/protobuf/src"
     ${TORCH_INCLUDE_DIRS})
 
-  if (onnxruntime_ENABLE_EAGER_MODE)
```

Comment:
Now, both eager mode and lazy tensor links against those libraries directly. Previously lazy tensor build didn't do so because the linker didn't complain.